components/automate-ui/src/app/page-components/search-bar-filter-bar/search-bar-filter-bar.component.ts

```diff
@@ -23,4 +23,9 @@ export class SearchBarFilterBarComponent {
   onRemoveFilterClick(filter: any): void {
     this.filterRemoved.emit({ detail: filter});
   }
+
+  trackBy(_index: number, filter: Chicklet): string {
```

Comment:
The `trackBy` in the corresponding template was never actually used because it did not have this in the code-behind.